sys/kern/src/arch/riscv32.rs

```diff
@@ -250,9 +254,9 @@ cfg_if::cfg_if! {
         use riscv::register::mtvec::{self, TrapMode};
 
         // Setup interrupt vector `mtvec` with vectored mode to the trap table.
-        // SAFETY: if _start_trap does not have the neccasary alignment,
-        // the address could become corrupt and traps will not jump to the
-        // expected address
+        /// # Safety
+        /// if `_trap_table` does not have the neccasary alignment, the address
+        /// could become corrupt and traps will not jump to the expected address
```

Comment:
Same comment as above. What is the responsibility on the caller to guarantee safety?

Reply:
The caller can't really do much, it's a question of if the linker(?) will align the _trap_table() to [where it should be](https://github.com/rivosinc/hubris/pull/59/files#diff-f6094245b6f04540f9c68fcf17feab28dfad806abfc7d604ec418481dc66ad84R273). I think marking this as safe as well makes more sense.